New-Win32Package.ps1

```diff
@@ -231,6 +231,7 @@ process {
                         Path        = "$([System.IO.Path]::Combine($AppPath, "Source"))\*"
                         Destination = $SourcePath
                         Recurse     = $true
+                        Exclude     = "Deploy-Application.ps1"
```

Comment:
This file is copied into the parent folder.
To make the contents of Files as small as possible, I have excluded this file.